### PR TITLE
Installing missing dependency for Hyperv-V Management Console (bnc#887944)

### PR DESCRIPTION
**Pull request by ovisan**

### CODE REVIEW

**Comment by vuntz** [comment]
Hrm, can you squash the three commits into one?

**Comment by vuntz** [comment]
+1

**Comment by tboerger** [comment]
A full bugzilla link in the description should be nice.
:+1:

File: chef/cookbooks/hyperv/attributes/default.rb [hunk 1]
```diff
@@ -6,6 +6,9 @@
   "Microsoft-Hyper-V" => {
     "restart" => false
   },
+  "Hyper-V-Tools" => {
```
Comment (vuntz) [inline chef/cookbooks/hyperv/attributes/default.rb]: Apparently, this needs to be changed to RSAT-Hyper-V-Tools-Feature. @ovisan can you confirm?

**Comment by ovisan** [comment]
Added the 2 dependencies manually instead of using /All switch from dism

File: chef/cookbooks/hyperv/attributes/default.rb [hunk 2]
```diff
@@ -6,8 +6,14 @@
...
   },
-  "Microsoft-Hyper-V-Management-PowerShell" => {
-    "restart" => true
+  "RSAT-Hyper-V-Tools-Feature" => {
+    "restart" => false,
+  },
+  "Microsoft-Hyper-V-Management-Clients" => {
+    "restart" => false,
+  },
+  "Microsoft-Hyper-V-Management-Powershell" => {
```
Comment (vuntz) [inline chef/cookbooks/hyperv/attributes/default.rb]: Typo: PowerShell, not powershell (I think)

Comment (ovisan) [inline chef/cookbooks/hyperv/attributes/default.rb]: displayed from dism is Powershell, but windows is case insensitive anyway. So this works as is.

**Comment by tboerger** [comment]
LGTM

**Pull request merged**